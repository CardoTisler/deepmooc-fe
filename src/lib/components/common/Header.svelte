<script>
	import ProfileIcon from '$lib/components/icons/ProfileIcon.svelte';
	import Placeholder from '$lib/components/Placeholder.svelte';
</script>

<header class="flex flex-row justify-between bg-white shadow-md mb-2">
	<a href="/main" class="w-64 pl-4"><Placeholder text="Logo" class="h-full" /></a>

	<div class="flex flex-row w-64 h-16">
		<div class="flex flex-col">
			<p>Eesnimi Perenimi</p>
			<a href="/" class="no-underline">Logi v√§lja</a>
		</div>
		<div class="ml-4 w-12">
			<ProfileIcon />
		</div>
	</div>
</header>

<style>
</style>
