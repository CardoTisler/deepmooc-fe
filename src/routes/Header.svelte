<script>
	import ProfileIcon from '$lib/components/icons/ProfileIcon.svelte';
</script>

<header class="flex flex-row justify-between bg-white shadow-md">
	<div class="flex w-32 h-16 bordered"><a href="/" class="w-full">Icon</a></div>

	<div class="flex flex-row w-64 h-16 bordered">
		<div class="flex flex-col">
			<p>Eesnimi Perenimi</p>
			<button class="self-end">Logi v√§lja</button>
		</div>
		<div class="bordered ml-4 w-12">
			<ProfileIcon />
		</div>
	</div>
</header>

<style>
</style>
