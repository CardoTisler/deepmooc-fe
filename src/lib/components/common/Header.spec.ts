import { render } from "@testing-library/svelte";
import Header from "./Header.svelte"

describe('Header.svelte', () => {
    it('should render component', () => {
        const { container } = render(Header);

        expect(container).toMatchSnapshot();
    });
});
