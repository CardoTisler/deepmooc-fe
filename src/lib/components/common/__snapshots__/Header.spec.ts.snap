// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header.svelte > should render component 1`] = `
<body>
  <div>
    <header
      class="bg-white shadow-md mb-2  dark:border-b-2 dark:border-b-yellow-300 s-TTyPr9WmAsat"
    >
      <div
        class="flex flex-row justify-between header dark:border-b-yellow-300 s-TTyPr9WmAsat"
      >
        <a
          class="w-64 pl-4 s-TTyPr9WmAsat"
          href="/main"
        >
          <img
            alt="Tooru logo"
            class="s-TTyPr9WmAsat"
            src="/tooru_logo.png"
          />
        </a>
         
        <div
          class="flex flex-row w-64 h-16 s-TTyPr9WmAsat"
        >
          <div
            class="flex flex-col s-TTyPr9WmAsat"
          >
            <p
              class="dark:text-yellow-300 s-TTyPr9WmAsat"
            >
              Eesnimi Perenimi
            </p>
             
            <a
              class="no-underline dark:text-yellow-300 s-TTyPr9WmAsat"
              href="/"
            >
              Logi v√§lja
            </a>
          </div>
           
          <div
            class="ml-4 w-12 s-TTyPr9WmAsat"
          >
            <svg
              fill="none"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="white"
                height="24"
                width="24"
              />
              <path
                clip-rule="evenodd"
                d="M2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12ZM11.9999 6C9.79077 6 7.99991 7.79086 7.99991 10C7.99991 12.2091 9.79077 14 11.9999 14C14.209 14 15.9999 12.2091 15.9999 10C15.9999 7.79086 14.209 6 11.9999 6ZM17.1115 15.9974C17.8693 16.4854 17.8323 17.5491 17.1422 18.1288C15.7517 19.2966 13.9581 20 12.0001 20C10.0551 20 8.27215 19.3059 6.88556 18.1518C6.18931 17.5723 6.15242 16.5032 6.91351 16.012C7.15044 15.8591 7.40846 15.7251 7.68849 15.6097C8.81516 15.1452 10.2542 15 12 15C13.7546 15 15.2018 15.1359 16.3314 15.5954C16.6136 15.7102 16.8734 15.8441 17.1115 15.9974Z"
                fill="#323232"
                fill-rule="evenodd"
              />
            </svg>
            <!--&lt;ProfileIcon&gt;-->
          </div>
        </div>
      </div>
    </header>
    <!--&lt;Header&gt;-->
  </div>
</body>
`;
